pkg/controller/defaults/placements.go

```diff
@@ -213,5 +213,41 @@ var (
 				},
 			},
 		},
+
+		"rgw-multi-instances": rook.Placement{
+			PodAntiAffinity: &corev1.PodAntiAffinity{
+				RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
+					corev1.PodAffinityTerm{
+						LabelSelector: &metav1.LabelSelector{
+							MatchExpressions: []metav1.LabelSelectorRequirement{
+								metav1.LabelSelectorRequirement{
+									Key:      "app",
```

Comment:
we can define key and value as a variable and use it, as its used in multiple places.

Reply:
@Madhu-1 , `metav1.LabelSelectorRequirement` is a `struct` not a `map[string]string`, so I am not sure how will I put an already defined key/value variable pair in their place.

Yes, I see that "app" string is used in lot of places, we could put that into a variable and use it in all the places...

Reply:
I meant to define a variable for app and rook-ceph-rgw. We can do it later that should not block this pr